src/libertem/udf/raw.py

```diff
@@ -26,8 +27,8 @@ class PickUDF(UDF):
     >>> result["intensity"].raw_data[0].shape
     (32, 32)
     '''
-    def __init__(self):
-        super().__init__()
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Hm, why this change? The `PickShiftedUDF` is subclassed from `UDF`, so `PickUDF` doesn't need to pass parameters through. If one wanted a common subclass to share code, it would be cleaner to have a `BasePickUDF` with the common parts, and derive both `PickUDF` and `PickShiftedUDF` from that, each with a final constructor. That way the parameters are clear for autocompletion etc.